### ISSUE TITLE
Web - Step 5 - No Langauge Set for "Verité" in Main Content Section of Page

### ISSUE THREAD

**Issue by jfc3-dol** [USDepartmentofLabor/Comply-Chain#64]
The word "Verité" for list items 9, 10, 11, and 12 in the "**Further Resources**" section should be marked up as French ( lang="fr" ). There might be other words that are not English on this page or others that need to be marked up too.

Step 5 - https://dolcontentdev.opadev.dol.gov/ilab/complychain/build/steps/5

**Comment by KyleLivingstonDOL** [comment]
Thanks. ZTI, can we do this? I can see if I can have our interns look at comply chain and mark words in the English version that are French or Spanish.

**Comment by jfc3-dol** [comment]
The same idea will apply to the Spanish and French version where there is English and the other language.

**Comment by Abychalil** [comment]
lang Attribute set to french at the mentioned places.
fixed

**Comment by jfc3-dol** [comment]
@Abychalil,

Thanks for correcting the language issues for French on the screen. Will look at it when the next version/build is sent, and I  have time to test.

**Comment by jfc3-dol** [comment]
Thanks for adding the language around the text "Verité" for the four list items.

I'm going to close this issue.
